media/css/mozorg/rise25.scss

```diff
@@ -0,0 +1,363 @@
+                    content: '';
+                    position: absolute;
+                    border-width: 10px;
+                    border-style: solid;
+                    border-color: transparent transparent #aaa;
+                    opacity: 0;
+                    left: 25%;
+                    top: 0;
+                    transform: translateY(-100%);
+                }
+
+                &::after {
+                    content: '';
+                    position: absolute;
+                    border-width: 9px;
+                    border-style: solid;
+                    border-color: transparent transparent #fff;
+                    opacity: 0;
+                    top: 0;
+                    left: calc(25% + 1px);
+                    transform: translateY(-100%);
+                }
+            }
+
+            .social-share-copied {
+                display: none;
+                opacity: 1;
+                left: -75%;
+
+                &::before {
+                    content: '';
+                    position: absolute;
+                    border-width: 10px;
+                    border-style: solid;
+                    border-color: transparent transparent #aaa;
+                    opacity: 1;
+                    top: 0;
+                    left: 25%;
+                    transform: translateY(-100%);
+                }
+
+                &::after {
+                    content: '';
+                    position: absolute;
+                    border-width: 9px;
+                    border-style: solid;
+                    border-color: transparent transparent #fff;
+                    opacity: 1;
+                    top: 0;
+                    left: calc(25% + 1px);
+                    transform: translateY(-100%);
+                }
+            }
+
+            @media #{$mq-md} {
+                .social-share-copied {
+                    &::before {
+                        bottom: $spacing-xs;
+                    }
+                }
+
+                &:hover,
+                &:active {
+                    .social-share-copy,
+                    .social-share-copy::before,
+                    .social-share-copy::after {
+                        opacity: 1;
+                    }
+                }
+            }
+        }
+
+        svg {
+            stroke: currentColor;
+        }
+    }
+}
+
+.nomination-section {
+    background: #f2f0e9;
+
+    .nomination-section-heading {
+        margin-right: auto;
+        margin-left: auto;
+        max-width: 80%;
+        text-align: center;
+
+        h2 {
+            @include font-base;
+        }
+    }
+}
+
+.nomination-grid {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: $spacing-xl;
+
+    @media #{$mq-md} {
+        grid-template-columns: 1fr 1fr;
+    }
+
+    @media #{$mq-lg} {
+        grid-template-columns: repeat(3, 1fr);
+    }
+}
+
+.nomination-card {
+    background-color: var(--card-color, #{$color-black});
```

Comment:
Suggestion (non-blocking): Is it worth adding a fallback `background-color` above this line for older browsers? They could probably all have the same color to make things simple? Without it there's some contrast issues:

![image](https://user-images.githubusercontent.com/400117/232019164-b5ce6017-035c-439d-90c8-3860ddf027ca.png)

Reply:
woof, yeah. Lets fix that